outbound/converter/interchange_translator.py

```diff
@@ -65,4 +63,7 @@ def __translate_edifact(self):
         return '\n'.join([segment.to_edifact() for segment in self.segments])
 
     async def __record_outgoing_state(self):
+        adaptor = get_persistence_adaptor(**{'table_name': 'nhais_state'})
```

Comment:
I think you can use the named argument syntax `table_name='nhais_state'` instead of unpacking the dict here